build-aux/bootstrap.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash -e
 
+# Needed to build GN itself.
+. /usr/lib/sdk/llvm12/enable.sh
+
+export LD_LIBRARY_PATH=/usr/lib/sdk/llvm12/lib:$LD_LIBRARY_PATH
```

Comment:
I can add such thing to enable.sh script if it helps. Does extension path need to be called first or ```LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib/sdk/llvm12/lib``` would work as well?